libsrc/c64/tgi/c64-hi.s

```diff
@@ -70,12 +70,9 @@ X2              := ptr3
 Y2              := ptr4
 TEXT            := ptr3
 
-ROW             := tmp2         ; Bitmap row...
-COL             := tmp3         ; ...and column, both set by PLOT
```

Comment:
What was wrong with the `ROW`and `COL` alias? I seems to me they helped to understand the code.

Reply:
It was used to check if LINE wants to plot inside the screen but that's a waste of resources since it is always called by tgi_clippedline() which already takes care of the screen boundaries.

Reply:
And BAR is already being clipped properly as well.